packages/zk-certificates/lib/registryTools.ts

```diff
@@ -33,14 +33,19 @@ export async function issueZkCert(
 
   const contractToCall = devnetGuardian ?? recordRegistry;
   // now we have the merkle proof to add a new leaf
-  const tx = await contractToCall.connect(issuer).addZkKYCRecord(
+  const tx = await contractToCall.connect(issuer).addZkCertificate(
```

Comment:
The revoke method in the other function below also needs to be changed.

Reply:
fixed